src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2417,6 +2419,16 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             $container->removeDefinition($classToServices[MercureTransportFactory::class]);
         }
 
+        if (ContainerBuilder::willBeAvailable('symfony/fake-chat-notifier', FakeChatTransportFactory::class, ['symfony/mailer'])) {
+            $container->getDefinition($classToServices[FakeChatTransportFactory::class])
+                ->replaceArgument('$mailerLocator', new ServiceLocatorArgument(new TaggedIteratorArgument('mailer', null, null, true)));
```

Comment:
I found out, that we did not tag our mailer's 🤔 

![CleanShot 2021-04-08 at 13 32 55](https://user-images.githubusercontent.com/995707/114020018-66e4e200-986f-11eb-87fb-e136748a2a4f.png)